tools/network_speed_test.py

```
@@ -17,10 +17,14 @@ def network_speed_test():
     except Exception as e:
         print(f"Error: {e}")
 
+def check_platform_compatibility():
+    supported = True
+    warnings = []
 
-def main():
-    network_speed_test()
-
+    try:
+        import speedtest
```

Ruff (0.11.9)

`speedtest` imported but unused

Remove unused import: `speedtest`

(F401)